libs/langchain/langchain/document_loaders/wikipedia.py

```diff
@@ -17,7 +17,7 @@ def __init__(
         self,
         query: str,
         lang: str = "en",
-        load_max_docs: Optional[int] = 100,
+        load_max_docs: Optional[int] = 25,
```

Comment:
why change default?

Reply:
@baskaryan It is described in the PR description ^